instance/models/instance.py

```diff
@@ -455,10 +455,8 @@ class AnsibleInstanceMixin(models.Model):
     """
     An instance that relies on Ansible to deploy its services
     """
-    ansible_source_repo_url = models.URLField(max_length=256,
-                                              default='https://github.com/edx/configuration.git')
-
-    configuration_version = models.CharField(max_length=50, default='master')
+    ansible_source_repo_url = models.URLField(max_length=256, blank=True)
+    configuration_version = models.CharField(max_length=50, blank=True)
```

Comment:
@smarnach @mtyaka Good point for the concept of deployment - it would definitely deserve its own class. I've added an entry in the uservoice about this: https://opencraft.uservoice.com/forums/280754-opencraft/suggestions/10727073-provisioning-history